src/selectLap/index.html

```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html>
-<html lang="en">
+<!doctype html>
```

Comment:
lowercase isn't wrong, but I think standard is uppercase for `DOCTYPE`. From what I can tell it doesn't influence anything and both forms are valid, so changing it is up to you.

Reply:
Strange, I don't think I changed this. Must've been prettier or something. Will change it back though.

Reply:
Yup, it's prettier. I'll just leave it as is